### DISCUSSION TITLE
10k download reached !

### DISCUSSION THREAD

**Discussion by MarianBulla** [OpenRadioss/OpenRadioss#2523] [Announcements]
Another milestone for OpenRadioss! 
The OpenRadioss Team are thrilled to announce we have reached 10 000 downloads !

Thank you all very much for using OpenRadioss !

Let's continue having fun with OpenRadioss  !

Thanks and best regards,
Marian

**Comment by Vortex-CAE** [comment]
Congratulations on the milestone Marian & thank you to the team.
